interview-questions.md

```diff
@@ -15,54 +15,54 @@ INSTRUCTOR EXAMPLE: What is a conditional statement?
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer:Git is the system used to move the data, whether getting, receiving. Github is where the data is stored. Its like a bank, github is like the safe that holds everyones money. However, the money must be accessed using transactions with a bankteller.
 
-  Researched answer:
+  Researched answer:Git is a manange system that allows you to track your source code. GitHub is a cloud- based service that allows you to store and manage Git repositories.
 
 
 
 2. Which JavaScript operators will return a Boolean value?
 
-  Your answer:
+  Your answer: I guess for loops.
 
-  Researched answer:
+  Researched answer: The logical (||) operator, is generally used with boolean values.
```

Comment:
A Boolean value will be returned when checking for a condition, which there are three types of operators that check the condition of a statement.